server/src/main/java/org/apache/cassandra/sidecar/handlers/livemigration/LiveMigrationStatusClearHandler.java

```diff
@@ -0,0 +1,111 @@
+import io.vertx.ext.auth.authorization.Authorization;
+import io.vertx.ext.web.RoutingContext;
+import org.apache.cassandra.sidecar.acl.authorization.BasicPermissions;
+import org.apache.cassandra.sidecar.cluster.instance.InstanceMetadata;
+import org.apache.cassandra.sidecar.concurrent.ExecutorPools;
+import org.apache.cassandra.sidecar.handlers.AbstractHandler;
+import org.apache.cassandra.sidecar.handlers.AccessProtected;
+import org.apache.cassandra.sidecar.livemigration.LiveMigrationStatusTracker;
+import org.apache.cassandra.sidecar.utils.CassandraInputValidator;
+import org.apache.cassandra.sidecar.utils.InstanceMetadataFetcher;
+import org.jetbrains.annotations.NotNull;
+
+import static org.apache.cassandra.sidecar.utils.HttpExceptions.wrapHttpException;
+
+/**
+ * Handler to clear the live migration status for an instance.
+ * <p>
+ * This endpoint removes the migration completion status, allowing the instance to be
+ * migrated again in the future. Clearing the status is necessary to prevent blocking
+ * future migrations of the same destination instance.
+ *
+ * <h3>Usage Safety:</h3>
+ * IMPORTANT: This endpoint should ONLY be called after completing the live migration
+ * process and the instance entry has been removed from the Live Migration map
+ * Calling this endpoint prematurely may lead to inconsistent migration state.
+ *
+ * <h3>Use Cases:</h3>
+ * <ul>
+ *   <li>After successful completion of a live migration to allow future migrations</li>
+ *   <li>To recover from erroneous COMPLETED status markings</li>
+ * </ul>
+ */
+public class LiveMigrationStatusClearHandler extends AbstractHandler<Void> implements AccessProtected
+{
+    private static final Logger LOGGER = LoggerFactory.getLogger(LiveMigrationStatusClearHandler.class);
+
+    private final LiveMigrationStatusTracker statusTracker;
+
+    @Inject
+    public LiveMigrationStatusClearHandler(InstanceMetadataFetcher metadataFetcher,
+                                           ExecutorPools executorPools,
+                                           CassandraInputValidator validator,
+                                           LiveMigrationStatusTracker statusTracker)
+    {
+        super(metadataFetcher, executorPools, validator);
+        this.statusTracker = statusTracker;
+    }
+
+    @Override
+    protected Void extractParamsOrThrow(RoutingContext context)
+    {
+        return null;
+    }
+
+    @Override
+    protected void handleInternal(RoutingContext context, HttpServerRequest httpRequest, @NotNull String host, SocketAddress remoteAddress, Void request)
+    {
+        InstanceMetadata instanceMetadata = metadataFetcher.instance(host);
+        statusTracker.clearMigrationStatus(instanceMetadata)
+                     .onSuccess(v -> {
+                         LOGGER.info("Successfully cleared live migration status for instance {}", host);
+                         context.response().setStatusCode(HttpResponseStatus.OK.code()).end();
```

Comment:
generally we want to provide some feedback to the client, especially if the end user is a person. I recommend the following, which is what we do consistently across the codebase
```suggestion
                         context.json(OK_STATUS);
```

Reply:
done, thanks for sharing the tip 👍